docs/ISSUE_TEMPLATE/feature_template.md

```diff
@@ -0,0 +1,24 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: "[FEATURE]: <your_feature>"
+labels: ":rocket: feature-request"
+assignees:
+---
+
+<!-- Provide a general summary of the issue in the Title above -->
+
+<!-- Before submitting an issue, please be sure to read the `CONTRIBUTING` document -->
+<!-- Then Put an `x` in the box bellow -->
+
+- [ ] **I have read the [CONTRIBUTING](https://github.com/berty/community/blob/master/CONTRIBUTING.md) document**
```

Comment:
I suggest to remove the checkbox and just add a comment with a link to the contributing file